Maquetado/scss/application.scss

```
@@ -0,0 +1,3 @@
+.title {
+    color: red;
+  }
```

Add EOF (End Of File) and remove the tabulation
```suggestion
}
```